src/components/SuggestedAction/SuggestedAction.tsx

```diff
@@ -57,85 +70,120 @@ const SuggestedAction = ({
   };
 
   return (
-    <Box
-      sx={{
-        padding: '8px 0 12px 0',
-        bgcolor: 'greyscale.100',
-        display: 'flex',
-      }}
-    >
-      <Box
-        sx={{
-          borderRadius: 2,
-          backgroundColor: setBorderColor(variant),
-          py: 0.5,
-          ml: 0.375,
-          width: 4,
-        }}
-      />
+    <>
+      <SuggestedActionHamburgerMenu
```

Comment:
this is the only part relevant to my PR in this file... idkk where the changes in the rest of the file came from 😕